client/src/components/ReportCollection.js

```diff
@@ -2,15 +2,12 @@ import React, {Component, PropTypes} from 'react'
 import {Button, Pagination} from 'react-bootstrap'
 import autobind from 'autobind-decorator'
 
-import LongActionModal from 'components/LongActionModal'
```

Comment:
LongActionModal is now not used anymore. Do you want to remove it or keep it for something else? If you do remove it,  rc-progress may also need removal from package.json